icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -265,6 +265,22 @@ class RepositoryConfig:
     def set_virtual_chunk_container(self, cont: VirtualChunkContainer) -> None: ...
     def clear_virtual_chunk_containers(self) -> None: ...
 
+class Diff:
+    @property
+    def new_groups(self) -> set[str]: ...
+    @property
+    def new_arrays(self) -> set[str]: ...
+    @property
+    def deleted_groups(self) -> set[str]: ...
+    @property
+    def deleted_arrays(self) -> set[str]: ...
+    @property
+    def updated_user_attributes(self) -> set[str]: ...
+    @property
+    def updated_zarr_metadata(self) -> set[str]: ...
+    @property
+    def updated_chunks(self) -> dict[str, int]: ...
```

Comment:
I would love to be able to visualize which chunks are changed instead of just how many

Reply:
Many customers have asked for this exact capability.

Reply:
Let's merge this and I'll add the richer info to `Diff`

Reply:
ill approve and add a ticket